CODEOWNERS

```
@@ -0,0 +1,44 @@
+# Core
+/ghcide @pepeiborra
+/ghcide/session-loader @pepeiborra @fendor
+/hls-graph @pepeiborra
+/hls-plugin-api @jneira @berberman
+/hls-test-utils @jneira
+/hie-compat @wz1000
+
+# Plugins
+/plugins/hls-alternate-number-format-plugin @drsooch
+/plugins/hls-brittany-plugin @fendor
+/plugins/hls-call-hierarchy-plugin @July541
+/plugins/hls-class-plugin @Ailrun
+/plugins/hls-eval-plugin @tittoassini
+/plugins/hls-explicit-imports-plugin @pepeiborra
+/plugins/hls-floskell-plugin @alanz @Ailrun
+/plugins/hls-fourmolu-plugin @georgefst
+/plugins/hls-haddock-comments-plugin @berberman
+/plugins/hls-hlint-plugin @jneira
+/plugins/hls-module-name-plugin @tittoassini
+/plugins/hls-ormolu-plugin @alanz
+/plugins/hls-pragmas-plugin @berberman @Ailrun
+/plugins/hls-qualify-imported-names-plugin @eddiemundo 
+/plugins/hls-refine-imports-plugin @rayshih 
+/plugins/hls-rename-plugin @OliverMadine
+/plugins/hls-retrie-plugin @pepeiborra
+/plugins/hls-splice-plugin @konn
+/plugins/hls-stylish-haskell-plugin @Ailrun
+/plugins/hls-tactics-plugin @isovector
+
+# Benchmarking
+/shake-bench @pepeiborra
+
+# Docs
+/docs @michaelpj @jneira
+
+# CI
+/.circleci @jneira @Anton-Latukha
+/.githhub @jneira @Anton-Latukha
```

There is a typo. BTW, I'm also willing to manage this, so:
```suggestion
/.github @jneira @Anton-Latukha @Ailrun
```